target/linux/realtek/dts-5.10/rtl8382_d-link_dgs-1210-28.dts

```diff
@@ -5,6 +5,24 @@
 / {
 	compatible = "d-link,dgs-1210-28", "realtek,rtl838x-soc";
 	model = "D-Link DGS-1210-28";
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio1 33 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	gpio1: rtl8231-gpio {
```

Comment:
Can we stop using the gpio1 label?
It doesn't really make sense, calling it just rtl8231 would be probably better as this way it looks like its just another GPIO controller from the CPU.

Reply:
Whatever we call it, we should keep it the same for every device. Either we rename all existing references before this PR or we keep gpio1 here as is and rename all of them afterwards.